testdata/crds/catalogd.operatorframework.io_catalogs.yaml

```diff
@@ -0,0 +1,172 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.11.4
+  name: catalogs.catalogd.operatorframework.io
+spec:
+  group: catalogd.operatorframework.io
+  names:
+    kind: Catalog
+    listKind: CatalogList
+    plural: catalogs
+    singular: catalog
+  scope: Cluster
+  versions:
+    - name: v1beta1
```

Comment:
Sorry about this, but I just merged in catalogd v0.3.0, which changed the CRDs. We should re-pull this one. I think it was just the version changing from v1beta1 to v1alpha1, but probably best to just copy the whole thing over again.